lua/test.sh

```diff
@@ -12,7 +12,7 @@ export VIS_THEME=theme
 if [ $# -gt 0 ]; then
 	test_files=$@
 else
-	test_files=$(find . -type f -name "*.in")
+	test_files="$(find . -type f -name "*.in") basic_empty_file.in"
```

Comment:
I meant the distinction between an empty file and one that doesn't exist, `vis file` when file doesn't exist is different to when file is empty.